### PR TITLE
Add NewRP

### PR DESCRIPTION
**Pull request by RealBenluka**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/newrp/manifest.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "nice_name": "New RolePlay",
+  "direct_ip": "newrp.de",
+  "server_wildcards": [
+    "%.newrp.de"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://newrp.de",
```
Comment (maaaathis) [inline minecraft_servers/newrp/manifest.json:12]: No real content; it's only the plesk default page

File: minecraft_servers/newrp/manifest.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "web_shop": "https://shop.newrp.de",
```
Comment (maaaathis) [inline minecraft_servers/newrp/manifest.json:13]: No content; it's only the plesk login page

File: minecraft_servers/newrp/background.png
Comment (maaaathis) [inline minecraft_servers/newrp/background.png:1]: Compress this file

File: minecraft_servers/newrp/background@2x.png
Comment (maaaathis) [inline minecraft_servers/newrp/background@2x.png:1]: Compress this file

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server not released yet; reopen PR once server is released

**Pull request closed without merging**